README.md

```diff
@@ -33,6 +33,15 @@ production-worthy implementation in 2021.
 Communications for Rust-Lightning and Lightning Development Kit happens through
 [LDK slack](http://lightningdevkit.org/).
 
+Crates
+-----------
+1. [lightning](./lightning)
+1. [lightning-background-processor](./lightning-background-processor)
```

Comment:
Maybe change the numbers :). But, should we say a brief word about each? Should be able to copy and tweak the information in each directory's `Cargo.toml`.

Reply:
I can change the numbers. I was also thinking about adding some description but didn't know where to look for it.

Reply:
Look at the `Cargo.toml` and the crate's documentation (ie the big `//!` comment at the top of `lib.rs`).

Reply:
I used the `Cargo.toml` docs. Let me know.